fileblock/src/inode.rs

```diff
@@ -11,7 +11,7 @@ enum FileType {
 struct Inode {
     inumber: u32,
     ftype: FileType,
-    fsize: u64,
-    block_location_id: u32,
+    fsize: u32,
+    block_location_ids: [u32;],
```

Comment:
right. so to clarify we are not talking here about the location _on disk_, e.g. which of the 512byte diskblocks a part of the file object is, but rather, which 4Kib block(s) hold the file data right? 

in that case it can be fixed sized at `1048576` right 1024 * 1024 because of the arbitrarily set limit of our disk (1gib)?

Reply:
Right, basically each I node is a fixed size structure that holds:

1. file meta data 
2. An array of pointers to disk where the data lives

The array of pointers should consume the remaining capacity left after you’ve defined the metadata.

If you use 256 bytes per inode, you can fit 16 inodes per I node block. That means the file system can store 16 * the number of blocks worth of files.